src/cube_fit/cube_fit.py

```diff
@@ -1,7 +1,7 @@
 from cube_fit.linalg import *
 
```

Comment:
```suggestion


```
two empty lines between imports and first function definitions is correct

Reply:
Ok